addons/mail/static/src/models/rtc_session/rtc_session.js

```diff
@@ -384,6 +364,13 @@ function factory(dependencies) {
         isMuted: attr({
             default: false,
         }),
+        /**
+         * Determine whether current session is either muted or deaf.
+         */
+        isMutedOrDeaf: attr({
```

Comment:
Need consistency, either:

infinitive verb: mute, deafen
preterite muted, deafened
adjective: mute, deaf

I'm not sure which one is best, but it must be one of those pair.